client/state/action-types.js

```diff
@@ -67,6 +67,8 @@ export const COMMENTS_CHANGE_STATUS_SUCESS = 'COMMENTS_CHANGE_STATUS_SUCESS';
 export const COMMENTS_COUNT_INCREMENT = 'COMMENTS_COUNT_INCREMENT';
 export const COMMENTS_COUNT_RECEIVE = 'COMMENTS_COUNT_RECEIVE';
 export const COMMENTS_DELETE = 'COMMENTS_DELETE';
+export const COMMENTS_DELETE_FAILURE = 'COMMENTS_DELETE_FAILURE';
+export const COMMENTS_DELETE_SUCCESS = 'COMMENTS_DELETE_SUCCESS';
```

Comment:
yeah please _never_ feel pressured to write code "just because it's in there somewhere" :wink:

these are specifically unwanted in the data layer